sys/include/phydat.h

```diff
@@ -98,6 +98,7 @@ enum {
     /* pressure */
     UNIT_BAR,       /**< Beer? */
     UNIT_PA,        /**< Pascal */
+    UNIT_HPA,       /**< hectoPascal */
```

Comment:
nack for this. Using uint `UNIT_PA` and a scale of `2` should be sufficient. The string conversion of this should yield `hPa` in any case with the change added below.